client/src/Components/HomePage/NewsForm/NewsForm.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import {NewsFormProps} from '../../../types/NewsFormProps';
+import { observer } from 'mobx-react-lite';
+import CustomView from '../../../CustomComponents/CustomView';
+import CustomText from '../../../CustomComponents/CustomText';
+import getThemeStore from '../../../stores/themeStore';
+import { darkMode, lightMode } from '../../colors/colors';
+
+const NewsForm : React.FC<NewsFormProps> = observer(({newsItem}) => {
+
+  return (
+    <CustomView
+    style={{
+        marginTop: 30,
+        marginLeft: 5,
```

Comment:
we should as much as we can avoid using margins and padding (not saying not to use it, but if we an do it using flex with space-between etc.. it's better

Reply:
DONE ✅